apps/files_sharing/lib/External/MountProvider.php

```diff
@@ -72,14 +72,14 @@ public function getMount(IUser $user, $data, IStorageFactory $storageFactory) {
 	}
 
 	public function getMountsForUser(IUser $user, IStorageFactory $loader) {
-		$query = $this->connection->prepare('
-				SELECT `remote`, `share_token`, `password`, `mountpoint`, `owner`
-				FROM `*PREFIX*share_external`
-				WHERE `user` = ? AND `accepted` = ?
-			');
-		$query->execute([$user->getUID(), 1]);
+		$qb = $this->connection->getQueryBuilder();
+		$qb->select('remote', 'share_token', 'password', 'mountpoint', 'owner')
+			->from('share_external')
+			->where($qb->expr()->eq('user', $qb->createNamedParameter($user->getUID())))
+			->andWhere($qb->expr()->eq('accepted', $qb->createNamedParameter(1)));
```

Comment:
```suggestion
			->andWhere($qb->expr()->eq('accepted', $qb->createNamedParameter(1, IQueryBuilder::PARAM_INT)));
```
Or is it a bool in the db?